std_execution.bs

```diff
@@ -5488,14 +5488,16 @@ template &lt;class E>  // arguments are not associated entities ([lib.tmpl-heads
                 <code><i>stop-src</i></code> refers to the stop source of
                 `sh_state`.
 
-            * Then, it checks to see if
-                <code><i>stop-src</i>.stop_requested()</code> is `true`. If so, it
-                calls `execution::set_stopped(out_r')`.
-
             * Otherwise, it adds a pointer to `op_state` to the list of
```

Comment:
Otherwise is dangling now. Should it be:
```suggestion
            * Then, it adds a pointer to `op_state` to the list of
```

Reply:
That's not how the Committee specifies these things. They're very prescriptive about wording. Things are specified like:

* If `X` is `true`, then _<case 1>_.
* Otherwise, if `Y` is `true`, then _<case 2>_.
* Otherwise, _\<fallback case>_.

Reply:
But that's now the case we are having. Currently, you have:
- If receiver contract has already been satisfied..
- Otherwise, it does X.
- Otherwise, it does Y. 

So, you seems to be describing the case if () else if () else. But it seems that the current wording is if () else () else (), which doesn't make sense to me.